data/services.json

```diff
@@ -32,19 +32,19 @@
         "description": "Your Entrance To The Blockchain World.",
         "color": "#0075FF",
         "supportEmail": "support@blocto.app",
-        "website": "https://blocto.io/"
+        "website": "https://blocto.portto.io"
       }
     },
     {
       "f_type": "Service",
       "f_vsn": "1.0.0",
       "type": "authn",
       "method": "EXT/RPC",
-      "uid": "fcw#authn",
+      "uid": "frw#authn",
       "endpoint": "chrome-extension://hpclkefagolihohboafpheddmmgdffjm/popup.html",
       "provider": {
         "address": "0x33f75ff0b830dcec",
-        "name": "Flow Ref",
+        "name": "Flow Reference Wallet",
```

Comment:
This won't work without UI changes, names get truncated to 10 chars currently

https://github.com/onflow/fcl-discovery/blob/09c73c329f81a233f37c81ebf8ac93bec35be0b8/components/ServiceCard.tsx#L115